src/tools/quest_parser/main.cpp

```diff
@@ -147,7 +150,7 @@ void createScript( boost::shared_ptr< Core::Data::Quest >& pQuestData, std::set<
 
    if( !pQuestData->itemReward0.empty() )
    {
-      rewards += "      static constexpr auto RewardItem[] = { ";
+      rewards += "      uint16_t RewardItem[] = { ";
```

Comment:
We have the type inferred by the compiler because if we ever change what we output, we don't have to manage the type there too - there's no real benefits to explicitly declaring the type